test/ttHConfiguration.py

```diff
@@ -30,7 +30,7 @@
         enable = cms.bool(True),
         categories_parameters = cms.PSet(
             # Per-category lepton pt cuts
```

Comment:
why?

Reply:
Well, I don't understand this commit. I don't remember I touched that intentionally :)